lib/fastly-rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module FastlyRails
-  VERSION = "0.6.0"
+  VERSION = "0.6.1"
```

Comment:
I would make this a 0.7.0 bump since we are adding new methods now. (I know I said the opposite yesterday but that was before the new features.)